<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSelectReferee</class>
 <widget class="QDialog" name="DlgSelectReferee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select umpire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,4,0">
     <item>
      <widget class="QLabel" name="laHeadline">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Match information</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout" columnstretch="5,1,5">
          <item row="1" column="0">
           <widget class="QLabel" name="laPair1Player1Team">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="laPair1Player1Name">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="laPair1Player2Name">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="laPair2Player1Name">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="laPair2Player1Team">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="laPair1Player2Team">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="laPair2Player2Name">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="laPair2Player2Team">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Umpire selection</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,1,2,4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Filter type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbFilterMode"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="laTeamSelection">
              <property name="text">
               <string>Team selection:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbTeamSelection"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="RefereeTableWidget" name="tabPlayers">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnSelect">
         <property name="text">
          <string>Use selected umpire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNone">
         <property name="text">
          <string>No umpire for this match</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RefereeTableWidget</class>
   <extends>QTableWidget</extends>
   <header>ui/DlgSelectReferee.h</header>
   <slots>
    <slot>onPlayerSelectionChanged()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFilterMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onFilterModeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTeamSelection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onTeamSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelect</sender>
   <signal>clicked()</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onBtnSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabPlayers</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onPlayerSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNone</sender>
   <signal>clicked()</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onBtnNoneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabPlayers</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>DlgSelectReferee</receiver>
   <slot>onPlayerDoubleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFilterModeChanged()</slot>
  <slot>onTeamSelectionChanged()</slot>
  <slot>onPlayerSelectionChanged()</slot>
  <slot>onBtnSelectClicked()</slot>
  <slot>onBtnNoneClicked()</slot>
  <slot>onPlayerDoubleClicked()</slot>
 </slots>
</ui>
